src/Util/PHP.php

```diff
@@ -7,25 +7,28 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
-
-namespace PHPUnit\Util\PHP;
+namespace PHPUnit\Util;
 
 use __PHP_Incomplete_Class;
 use ErrorException;
+use SebastianBergmann\Environment\Runtime;
 use PHPUnit\Framework\Exception;
 use PHPUnit\Framework\TestResult;
 use PHPUnit\Framework\TestFailure;
 use PHPUnit\Framework\Test;
 use PHPUnit\Framework\SyntheticError;
-use PHPUnit\Util\InvalidArgumentHelper;
-use SebastianBergmann\Environment\Runtime;
 
 /**
- * Utility methods for PHP sub-processes.
+ * Default utility for PHP sub-processes.
  */
-abstract class AbstractPhpProcess
+class PHP
```

Comment:
Name is too generic, when you give only a name to someone, he will have on idea what to expect.
for logic in this class - maybe replace most of it with symfony/process ?

Reply:
See #1342. But that should not happen before #2015.

Reply:
:+1: for switching to process component in separated PR/issue.
Yet, still, `PHP` is way too generic and meaningless name, it shall be modified